### ISSUE TITLE
Question Regarding Training

### ISSUE THREAD

**Issue by emperorbrutus** [Thanos-DB/FullyConvolutionalTransformer#27]
Hello! I hope that you are still able to have a discussion regarding your repository.

As for now, I have tried to use your random splits mentioned in #6, tried not to change anything from your repo, and tried to start the training myself, but I wasn't able to reproduce your results (I achieved significantly lower results compared to your notebook). May I have your solution from this matter? I was expecting that the problem comes from the differences between the libraries version installed on my device and yours, but I would like to hear from you. Thanks in advance!

**Comment by Thanos-DB** [comment]
Hi @emperorbrutus,

how lower? If you have not changed anything i would also guess it is because of the different versions. Can use the docker image? With it you should get something very close.

**Comment by emperorbrutus** [comment]
Hello. Thank you for replying!

My last training session achieved the score [0.8204 0.8055 0.7015] and 0.7758 as the average score. The score was quite similar compared to my other training session when I tried to use the actual test set given in the dataset (Not using a portion from the train set as the test set). What makes it more confusing is I have tried to use the same library versions from your repo below:
- tensorflow==2.8.0 (I can't download the 2.5.0 version, so I decided to install other version)
- keras==2.8.0
- matplotlib==3.3.2
- medpy==0.4.0
- nibabel==3.2.1
- opencv-python
- scikit-image
- scikit-learn

As for the Docker Image, I haven't had any experience using that software. By the way, I ran the training in Google CoLab. Could it also impact the performance of the model and the overall result? Thanks!

**Comment by Thanos-DB** [comment]
Hi @emperorbrutus,

can you try with my split first to see if you get the same metrics? This will tell us if the used versions are ok.
I have heard from others that they get lower scores for the official test set but i have not tried it myself as when i was doing it the ground truths were not available, you had to upload the predictions to the evaluation server (which does not exist anymore). 
I would say, first try my split, if this does not work try to make the docker image work.

I do not have any experience with colab, i tried googling it and apparently others experience the same. Maybe try a google search and see if any recommendation helps.

- Thanos

**Comment by emperorbrutus** [comment]
Okay. I'll try to apply your recommendations. Thank you for your reply!

**Comment by emperorbrutus** [comment]
Hi, @Thanos-DB . It's me again. May I ask which Python version did you use for training the model? Thanks!
